app/src/main/kotlin/com/wire/android/ui/home/conversations/messages/item/MessageBubbleItem.kt

```diff
@@ -120,21 +122,16 @@ fun MessageBubbleItem(
                 val shape = RoundedCornerShape(dimensions().corner16x)
 
                 val bubbleColorOther: Color = if (messageStatus.isDeleted) {
-                    MaterialTheme.colorScheme.surface
+                    colorsScheme().surface
                 } else {
-                    MaterialTheme.colorScheme.background
+                    colorsScheme().otherBubble.primary
                 }
                 val bubbleColor = when {
                     messageStatus.isDeleted || message.decryptionFailed -> {
-                        MaterialTheme.colorScheme.surface
+                        colorsScheme().surface
                     }
 
-                    isSelfMessage -> {
-                        MaterialTheme.wireColorScheme.wireAccentColors.getOrDefault(
-                            accent,
-                            MaterialTheme.wireColorScheme.primary
-                        )
-                    }
+                    isSelfMessage -> accent.primary()
```

Comment:
In `colorsScheme()` we already have colors updated according to currently selected accent, right? So why `accent.primary()` and not just `colorsScheme().primary`?